### ISSUE TITLE
Confusing error message whe missing an 'end'

### ISSUE THREAD

**Issue by RandomString123** [JuliaEditorSupport/julia-vscode#461]
I don't know if this is a LanguageServer problem, a Julia problem, or a me problem but posting it to get some feedback.   

In most IDEs / Languages when there is a problem parsing a file due to a missing closing token it is handled by saying expected '}' or whatever the closing identifier is.  When the LanguageServer runs into this issue it puts out a message that just says 'Parsing Failure'.  It took me a while to equate 'Parsing Failure' with expected 'end'.

**Comment by ZacLN** [comment]
Yep this is a work in progress, I've quickly added https://github.com/JuliaEditorSupport/LanguageServer.jl/pull/259 which should provide a bit more information

**Comment by RandomString123** [comment]
Great!  Thanks for a quick turnaround.
